caikit/core/data_model/dataobject.py

```diff
@@ -186,19 +186,21 @@ def decorator(cls: _DataObjectBaseT) -> _DataObjectBaseT:
         # If it's not an enum, fill in any missing field defaults as None
         # and make sure it's a dataclass
         if not issubclass(cls, Enum):
-            log.debug2("Wrapping data class %s", cls)
+            # alog needs a stub file or some method of typing the monkey-patched methods.
```

Comment:
This would indeed be a nice improvement to [alog](https://github.com/IBM/alchemy-logging/issues/new/choose) 😉!

Reply:
I tried it once about 6 months ago.  :-/  Maybe I still have the pyi file.  I recall it was tricky.

Reply:
In base.py I just continues with `# type: ignore`.  I'll see if I find time to do a PR on alog.  I did look for my old effort and could not locate it.  Might be on my old laptop.